Apps/WebClient/src/ClientApp/src/constants/errorType.ts

```diff
@@ -38,4 +38,6 @@ export enum ErrorSourceType {
     ExportRecords = "export record",
     Notification = "notification",
     OrganDonorRegistration = "organ donor registration",
+    DiagnosticImaging = "diagnostic imaging exam",
+    PatientData = "patient data",
```

Comment:
A case for PatientData should be added to `pluralizeErrorSourceType()` in `utility/errorTranslator.ts` so the displayed error message will be "Unable to retrieve patient data" rather than "Unable to retrieve patient datas".  (The default pluralization just appends an 's'.)